cpp/oneapi/dal/io/csv/detail/graph_service.hpp

```diff
@@ -1,4 +1,4 @@
-/* file: common.hpp */
+/* file: graph_service.hpp */
 /*******************************************************************************
```

Comment:
Why common is renamed? We must preserve `common.hpp` in `dal/io` even it's empty. This is a part of public API.
BTW, it will include `"oneapi/dal/common.hpp"`

`/* file: common.hpp */` can be removed, we no longer require filenames in the files.

Reply:
It is renamed because it contains only graph specific structures